Grammar/python.gram

```diff
@@ -940,8 +940,18 @@ fstring_format_spec[expr_ty]:
 fstring[expr_ty]:
     | a=FSTRING_START b=fstring_middle* c=FSTRING_END { _PyPegen_joined_str(p, a, (asdl_expr_seq*)b, c) }
 
+tstring_replacement_field[expr_ty]:
+    | '{' a=annotated_rhs debug_expr='='? conversion=[fstring_conversion] format=[fstring_full_format_spec] rbrace='}' {
+        _PyPegen_interpolation(p, a, debug_expr, conversion, format, rbrace, EXTRA) }
+    | invalid_replacement_field
+tstring_middle[expr_ty]:
+    | tstring_replacement_field
+    | t=FSTRING_MIDDLE { _PyPegen_constant_from_token(p, t) }
```

Comment:
I find it a bit odd to have now FSTRING_MIDDLE. I don't mind but maybe we can rename it to STRING_CHUNK or something common. I don't care a lot if you don't like the idea

Reply:
Yeah, you're right. Done now!